Makefile

```diff
@@ -51,6 +53,12 @@ bindings: $(PKG_BINDINGS_DIR)/nvml.h $(SOURCES) | $(PKG_BINDINGS_DIR)
 	cd -> /dev/null
 	rm -rf $(PKG_BINDINGS_DIR)/types.go $(PKG_BINDINGS_DIR)/_obj
 
+SED_SEARCH_STRING = WARNING: This file has automatically been generated on
+SED_REPLACE_STRING = WARNING: THIS FILE WAS AUTOMATICALLY GENERATED.
```

Comment:
Does it make sense to use `:=` here since we don't need these vars to be regenerated based on future variable definitions.

Reply:
Sure. Will update as part of the rebase.